### ISSUE TITLE
Keep maps up to date

### ISSUE THREAD

**Issue by dew54** [mapsme/omim#11883]
Hello, is there a way to keep maps on my app (maps and app build myself) up to date? is there a way to apply a sort of change file and update the maps? 
Thanl you, 
Davide

**Comment by dew54** [comment]
Hello mpimenov, thanks for your quick reply!
Point 3 is the most interesting.. please, could you link me something where i could start read from?
Thank you very much.. 
Bye! Davide

**Comment by mpimenov** [comment]
I would start with forking the repository and skimming the latest commits. For example, see this PR: https://github.com/mapsme/omim/pull/11678/files

The most relevant parts are generator/, storage/, and indexer/.

**Comment by maksimandrianov** [comment]
The easiest way to generate maps  is described here: https://github.com/mapsme/dockerfiles/blob/master/maps_generator/README.md

You can also control the building of generation tools and generate maps manually:
https://github.com/mapsme/omim/blob/master/tools/python/maps_generator/README.md

**Comment by dew54** [comment]
> The easiest way to generate maps is described here: https://github.com/mapsme/dockerfiles/blob/master/maps_generator/README.md
> 
> You can also control the building of generation tools and generate maps manually:
> https://github.com/mapsme/omim/blob/master/tools/python/maps_generator/README.md

Yeah, i've already created my maps, i wonder if there's a way to update maps instead of generate a new one every time!

**Comment by mpimenov** [comment]
It depends on your exact goals.
You still need to generate a new map that you want to update to, and if it all happens locally on your machine there is not much sense in using mwm_diff.

**Comment by dew54** [comment]
I'm saying that it'll be great if i can keep the map updated by downloading only changes from my  own map server, if this is not possible i think i will need to generate the entire map again with changes and then re-download it.

**Comment by mpimenov** [comment]
Our process requires you to generate the new map on your server in any case.
Then, if you so wish, you can generate a diff file to update from one version to another. But before trying this, please make sure you are comfortable with updating without diffs.
